### DISCUSSION TITLE
Venteliste: Vagten går ikke til næste, når den afvises.

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#37] [Bugs] [answered]
Fra Malene: 

Der er noget galt med ventelisten. Den tager ikke den næste på listen.

Situationen var at jeg i går fik et afbud, som jeg gik ind og slettede, derefter tog den automatisk den første på ventelisten op, hende ringede jeg til og hun afslog vagten via den automatiske mail hun fik. Men nu sender den ikke en mail til den næste der er på ventelisten?

Jeg håber det kan blive rettet forholdsvis hurtigt, da vi går ind i 6 uger, hvor dette vil opstå mange gange.

![image](https://github.com/xooom-community/product/assets/167326849/f31710cd-88f5-484f-bd9c-2fd6f8a099d2)

**Comment by mikkelsl** [comment]
@antonlund18
Hvis du kan klemme det ind, vil du lave et issue i Development på den her, og se om du kan finde noget, der virker uhensigtsmæssigt ifm. venteliste? Vi kan bare lige tage en snak om opsætningen, når det passer dig.

Vi kører et cronjob hver 10. minut, som kører efter planen, kan ses i Administration under Cronjobs på dashboard.

**Comment by mikkelsl** [comment, marked as answer]
Vi fortsætter her: https://github.com/orgs/xooom-community/projects/1/views/1?pane=issue&itemId=64679138
